waves_core/celery.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import, unicode_literals
+from celery import Celery
+import os
+
+import django
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "waves_core.settings")
+django.setup()
+
+app = Celery('waves_core', broker_pool_limit=1, broker='redis://localhost:6379', result_backend='redis://localhost:6379')
```

Comment:
This should use at least some settings.py variable instead of Hardcoded broker uri.

Reply:
You're right. In the most recent commit, I have added the celery broker url to the conf file of waves-core.